examples/hotrod/cmd/customer.go

```diff
@@ -35,7 +36,7 @@ var customerCmd = &cobra.Command{
 		zapLogger := logger.With(zap.String("service", "customer"))
 		logger := log.NewFactory(zapLogger)
 		server := customer.NewServer(
-			net.JoinHostPort(customerOptions.serverInterface, strconv.Itoa(customerOptions.serverPort)),
+			net.JoinHostPort(customerOptions.serverInterface, strconv.Itoa(config.CustomerPort)),
```

Comment:
What about customerOptions.serverPort? If it's no longer used anywhere we should delete it from the struct.